QandA/Cohort-2-Questions.md

```diff
@@ -0,0 +1,39 @@
+Chapter 2 Questions: 
+- For exercise 2, section 2.5.1, why do I get one less copy when running code?
+    - It could be a difference between R and Rstudio. You also may need to switch your environment to something other than global. 
+- How does R allocate memory for different length character strings? 
+    - Seems to be allocating bytes in increments of ~8 characters (https://youtu.be/pCiNj2JRK50?t=988) 
+- Why don’t lists with repeated values that aren’t characters point to the same object, like character vectors do? (https://youtu.be/pCiNj2JRK50?t=2916) 
+- If you have to run garbage collector, are you doing something wrong with your code? (https://youtu.be/pCiNj2JRK50?t=1686) 
+- How is removing objects dealt with in the garbage collection process? (https://youtu.be/pCiNj2JRK50?t=1811) 
+- When might you want to use local? (https://youtu.be/pCiNj2JRK50?t=1869) 
+- Why does the example in section 2.5.1 make 3 copies when its a data frame? (https://youtu.be/pCiNj2JRK50?)t=2096) 
+- Where can I learn more about copy behavior with primitive functions? (https://youtu.be/pCiNj2JRK50?t=2587) 
```

Comment:
You can add these to `02-names_and_values.Rmd` where I made `<h2>` headers for each section and annotate questions and answers as:

```
## 2.5.1.2 Exercise {-}

:::question
Why do I get one less copy when running code?
:::

:::TODO
:::
```

[replacing TODO with the answer if answered!

...That said I think this specific question is already answered in the text but that should provide a template 😄